tests/test_kernels/test_stationary.py

```diff
@@ -241,6 +242,44 @@ class TestRationalQuadratic(BaseTestKernel):
     default_compute_engine = DenseKernelComputation()
 
 
+@pytest.mark.parametrize(
+    "kernel",
+    [RBF, RationalQuadratic, Matern12, Matern32, Matern52],
+)
+@pytest.mark.parametrize("tswitch_param", [0.1, 0.25, 0.5, 0.75, 0.9])
+def test_changepoint_kernel(kernel: AbstractKernel, tswitch_param: float) -> None:
+    n_kerns = 2
```

Comment:
This looks good. The only thing is, I would like to see (a) an error is thrown if `n_kerns` is wrong and this is tested.

Reply:
Sorry to ask basic question, how does write a test to make sure that a proper error is thrown?